pkgs/by-name/da/davis/package.nix

```diff
@@ -16,9 +16,10 @@ php.buildComposerProject (finalAttrs: {
     hash = "sha256-Zl+6nrgspyg6P9gqYwah81Z6Mtni6nUlCp4gTjJWn9M=";
   };
 
-  vendorHash = "sha256-ZV5GNNtex+yKaMP5KaQkx5EaJRAJSwJjIZOCcXlnVW4=";
+  vendorHash = "sha256-pCWM1kgk30Au9i8TflrmZXw/PJmo3tdW565BXwncsZU=";
 
   postInstall = ''
+    chmod -R u+w $out/share
```

Comment:
Otherwise we have a permission denied. It's similar to https://github.com/NixOS/nixpkgs/blob/91b02440a51ceea54ee5ca61b2d018f41c22695f/pkgs/servers/nextcloud/packages/apps/hmr_enabler.nix#L25

I'm not sure if need to remove write after the operation is complete.

Reply:
It's only needed if files have to be moved between directories. So far we have only 4 examples in nixpkgs, so I'm not sure.

Reply:
Hm, I'm not qualified enough to comment on that as I don't know too much about the composer builders. What process gets the permission denied error and when does it happen?

Reply:
mv and rm as on $out is set to read-only during install.